лабораторная4/task1.py

```diff
@@ -0,0 +1,107 @@
+class Earphones:
+    """
+            Создание и подготовка к работе объекта "Наушники"
+            :param name: Марка, защищенный аргумент, пользователь не может создать новые виды наушников
+            :param manufacturer: Производитель, защищенный аргумент, пользователь не может создать новые виды наушников

```

Comment:
```suggestion
            :param name: Марка, защищенный аргумент
            :param manufacturer: Производитель, защищенный аргумент
```
Если есть сеттер, то можно создавать новые экземпляры классов, ну или удалить setter